packages/webpack-config/src/webpackConfig.ts

```diff
@@ -20,7 +20,10 @@ import type {
   WebpackPluginFunction,
   WebpackPluginInstance,
 } from "webpack";
-import { BannerPlugin, DefinePlugin } from "webpack";
+// eslint-disable-next-line import-x/no-named-as-default
+import webpack from "webpack";
+
+const { BannerPlugin, DefinePlugin } = webpack;
```

Comment:
❓ Any idea why the direct import didn't work? If yes, could you add a brief explanation as a comment?

Reply:
It has something to do with the CS <-> ESM boundary, but I couldn't determine the exact cause. I just wanted to have this as a stop-gap until someone takes the time to upgrade a non-React web application to ViteJS to show how that should be done and then that will be the way moving forward.